website/docs/dependencies/0040-install-git-repo.md

```diff
@@ -36,6 +37,7 @@ In this example, VC clones https://github.com/eembc/coremark.git into the runtim
       "Type": "GitRepoClone",
       "Parameters": {
           "RepoUri": "https://github.com/eembc/coremark.git",
+          "Checkout": "eefc986"
```

Comment:
Thanks, as part of this I would propose you add the Checkout to workloads like coremark. Find a good release commit